content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -1,42 +1,45 @@
-- [`metrics`](/concepts/components/prisma-client/metrics) since release [3.15.0](https://github.com/prisma/prisma/releases/tag/3.15.0) - [Submit feedback](https://github.com/prisma/prisma/issues/13579)
-- [`orderByNulls`](/concepts/components/prisma-client/filtering-and-sorting#sort-with-null-records-first-or-last) since release [4.1.0](https://github.com/prisma/prisma/releases/tag/4.1.0) - [Submit feedback](https://github.com/prisma/prisma/issues/14377)
-- [`tracing`](/concepts/components/prisma-client/opentelemetry-tracing) since release [4.2.0](https://github.com/prisma/prisma/releases/tag/4.2.0) - [Submit feedback](https://github.com/prisma/prisma/issues/14640)
-- [`filteredRelationCount`](/concepts/components/prisma-client/aggregation-grouping-summarizing#filter-the-relation-count) since release [4.3.0](https://github.com/prisma/prisma/releases/tag/4.3.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15069)
-- [`fieldReference`](/reference/api-reference/prisma-client-reference#compare-columns-in-the-same-table) since release [4.3.0](https://github.com/prisma/prisma/releases/tag/4.3.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15068)
-- [`multiSchema`](https://github.com/prisma/prisma/issues/1122#issuecomment-1231773471) since release [4.3.0](https://github.com/prisma/prisma/releases/tag/4.3.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15077)
-- [`postgresqlExtensions`](/concepts/components/prisma-schema/postgresql-extensions) since release [4.5.0](https://github.com/prisma/prisma/releases/tag/4.5.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15835)
-- [`deno`](/guides/deployment/deployment-guides/deploying-to-deno-deploy) since release [4.5.0](https://github.com/prisma/prisma/releases/tag/4.5.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15844)
-- [`extendedWhereUnique`](/reference/api-reference/prisma-client-reference#filter-on-non-unique-fields-with-userwhereuniqueinput) since release [4.5.0](https://github.com/prisma/prisma/releases/tag/4.5.0) - [Submit feedback](https://github.com/prisma/prisma/issues/15837)
-- [`clientExtensions`](/concepts/components/prisma-client/client-extensions) since release [4.7.0](https://github.com/prisma/prisma/releases/tag/4.7.0) - [Submit feedback](https://github.com/prisma/prisma/issues/16500)
-- [`views`](/concepts/components/prisma-schema/views) since release [4.9.0](https://github.com/prisma/prisma/releases/tag/4.9.0) - [Submit feedback](https://github.com/prisma/prisma/issues/17335)
-- `jsonProtocol` since release [4.11.0](https://github.com/prisma/prisma/releases/tag/4.11.0) - [Submit feedback](https://github.com/prisma/prisma/issues/18095)
+| Feature                                                                                                                         | Released into Preview                                          |                          Feedback issue                          |
```

Comment:
"Feedback Issue" is not actually centered in the rendered output, but the "Submit feedback" links are.